boards/arm/nrf9160_pca10090/nrf9160_pca10090.yaml

```diff
@@ -12,3 +12,4 @@ supported:
   - i2c
   - pwm
   - watchdog
+  - counter
```

Comment:
I wonder if this could be added also to the _NS board

Reply:
Just to be correct - the two board definitions (S , NS) ideally should be able to execute all counter tests

Reply:
it should but enabling ns board in vanilla zephyr does not may much sense since there is no means to run it without spm.